src/lint.ts

```diff
@@ -97,10 +97,17 @@ export const lintAction = async (
         );
         continue;
       }
+
+      /*
+       * base can be override by compareFrom
+       * if neither base nor compareFrom are given use defaultBranchName
+       */
       const base =
         options?.compareFrom ??
+        branchName ??
```

Comment:
We shouldn't have two ways of passing the same data. Either remove it from options or remove the `branchName` param and always use the value from options.

Reply:
I see your point. 
I think I'll keep the base param and just remove the compareTO in that case.
Reason: compatiblity with "sweater-comb lint" command

I'll updatete the orb as well